doc/guide/guide.rst

```diff
@@ -17,11 +17,10 @@ Users Guide
    guide-steady.rst
    guide-piqs.rst
    guide-correlation.rst
-   guide-control.rst
    guide-bloch.rst
    guide-visualization.rst
-   guide-parfor.rst
    guide-saving.rst
    guide-random.rst
    guide-settings.rst
    guide-measurement.rst
+   guide-control.rst
```

Comment:
Should there be a guide-qip placeholder that links to `qutip-qip` the way `guide-control` points users to `qutip-qtrl` and `qutip-qoc`?